internal/replication/tables.go

```diff
@@ -176,13 +183,8 @@ func RenameSubscriptionTable(db *sql.DB, publicationName string, table PgTable)
 	return err
 }
 
-func CheckSubscriptionTableDetail(db *sql.DB, table PgTable) error {
-	subscriptionTable := PgTable{
-		Schema: table.Schema,
-		Name:   table.Name,
-	}
-
-	err := tableColumns(db, &subscriptionTable, "")
+func CheckSubscriptionTableDetail(db *sql.DB, table PgTableDetail) error {
+	subscriptionTable, err := tableColumns(db, PgTable{Schema: table.Schema, Name: table.Name}, "")
```

Comment:
Can the `sqlJoin` argument be replaced with a boolean? I see that there is only a one join used. Will there be others that would warrant the use of custom SQL passing. Otherwise this might get flagged.